packages/server/src/index.ts

```diff
@@ -22,6 +23,17 @@ export async function main(configName: string): Promise<void> {
   const server = app.listen(config.port);
   server.keepAliveTimeout = config.keepAliveTimeout ?? 90000;
   globalLogger.info('Server started', { port: config.port });
+  gracefulShutdown(server, {
+    timeout: config.shutdownTimeoutMilliseconds,
+    development: process.env.NODE_ENV !== "production",
```

Comment:
@codyebberson might be nice if the config defined environment with an enum of sorts🤔